src/main/resources/db/migration/V21__POL-650_org_id_constraints.sql

```diff
@@ -0,0 +1,13 @@
+DROP INDEX pol_cus_nam_idx;
+
+ALTER TABLE policy
+    DROP CONSTRAINT policy_pkey,
+    ALTER COLUMN customerid DROP NOT NULL,
```

Comment:
isn't the  customer_id the account_id field? Are we removing it?

Reply:
Yes, this is where the account ID is stored, but this script only removes a `NOT NULL` constraint, not the field itself.